input/drivers_hid/wiiusb_hid.c

```diff
@@ -345,8 +390,33 @@ static int wiiusb_hid_add_adapter(void *data, usb_device_entry *dev)
    RARCH_LOG("Device 0x%p attached (VID/PID: %04x:%04x).\n",
          adapter->device_id, desc.idVendor, desc.idProduct);
 
-   wiiusb_hid_device_add_autodetect(adapter->slot,
-         device_name, wiiusb_hid.ident, desc.idVendor, desc.idProduct);
+   // Retrode support
```

Comment:
Still some C++ comments here. C comments are what is needed across the entire codebase (at least for C files).

Reply:
Fixed.

Reply:
Fixed. Sorry, overlooked.